### ISSUE TITLE
The mastery completion sign updates only after a page refresh and not real time.

### ISSUE THREAD

**Issue by harishbalachandran** [learningequality/kolibri#1680]
## Summary

A learner had completed and came out of the exercise and found the green completed tick did not get updated real time, but after refreshing the page the completed tick appeared. 

## System information
 - Version: Kolibri 0.4.0beta10
 - Operating system: Ubuntu 14.04 LTS
 - Browser: Chrome

## How to reproduce
1. Attempt an exercise or master it.
2. Come out of the exercise.
3. The completed or In progress stamp is not updated real time.

## Screenshots
Learner has mastered the topic.
![whatsapp image 2017-06-16 at 8 53 16 pm](https://user-images.githubusercontent.com/13453618/27233432-10160d78-52d7-11e7-847a-841e87703bc8.jpeg)

He exited the exercise and the completed sign on the thumbnail is not update:
![whatsapp image 2017-06-16 at 8 53 16 pm 1](https://user-images.githubusercontent.com/13453618/27233430-0ec0ba9a-52d7-11e7-9560-a942c1cd221d.jpeg)

But on refreshing the page the thumbnail has the completed sign.

**Comment by jonboiser** [comment]
@harishbalachandran This might be a duplicate of #1656.

When you are with this learner, could you check to see if their points are updated temporarily when the "good work" pop-up appears, then get reverted after the "good work" pop-up is dismissed?

**Comment by harishbalachandran** [comment]
> When you are with this learner, could you check to see if their points are updated temporarily when the "good work" pop-up appears, then get reverted after the "good work" pop-up is dismissed?

This was a problem I had reported before, but now this issue of points reverting is not observed, points get updated and stays the same even after dismissing the popup.

**Comment by christianmemije** [comment]
Fixed in #1748
